nncf/quantization/algorithms/min_max/onnx_backend.py

```diff
@@ -154,16 +154,14 @@ def create_convert_insertion_command(
         raise nncf.InternalError("FakeConvert insertion not implemented in ONNX backend!")
 
     @staticmethod
-    def unify_statistics(
-        statistics: List[ONNXMinMaxTensorStatistic],
-    ) -> ONNXMinMaxTensorStatistic:
+    def unify_statistics(statistics: List[MinMaxTensorStatistic]) -> MinMaxTensorStatistic:
```

Comment:
This method can be placed in the common algorithm part due to unification.

Reply:
good suggestion, done